### ISSUE TITLE
Please add the emoji poll feature to the elementor form.Please check. Very important

### ISSUE THREAD

**Issue by 79ho3ein** [elementor/elementor#14991]
It would be great if we could use the elementor form to get users' emotions in the form of emojis.
It is better to be able to upload emojis as desired and display options for the user.For example :
Food quality : 😡😐😃😋

And other cases where the user can select one of these 4 emojis to convey her feelings.

Please check this feature. Excellent and unique.
thanks

**Comment by fabianenzensperger** [comment]
Emoji's and picture's. Like a quiz. 😉

**Comment by 79ho3ein** [comment]
@fabianenzensperger 
Exactly

**Comment by promocot** [comment]
will be better if adding an option with choose emoji or stars, number, word (I mean, the possibility to create feedback form) add full customize, color, size, custom SVG upload

**Comment by 79ho3ein** [comment]
Yes, that's right.Adding a star-studded poll feature to the elementor form is also a great idea that you mentioned.

Thank you for your attention.
